content/guides/ethereum/concepts/_index.md

```diff
@@ -6,8 +6,8 @@ protocol: ETH
 release: beta
 ---
 
-In this section, you will find the list of dfuse concepts that apply specifically to the EOSIO
-chains supported by dfuse APIs.
+In this section, you will find the list of dfuse concepts that apply specifically to the ETHEREUM
+chain supported by dfuse APIs.
```

Comment:
J'pense qu'on peut garder le pluriel pour chains, puisqu'on supporte Rinkey, Mainnet et éventuellement d'autres chains. Le concept est valide pour toutes les chains Eth. 

Aussi, on va garder la consistency par rapport aux caps re: Ethereum.. EOSIO est tjrs caps, mais Eth c'est pas caps ailleurs.

Keep 'em coming!

Reply:
Agreed. Will close the pull request and open a new one with the correct changes.